### DISCUSSION TITLE
other solutions

### DISCUSSION THREAD

**Discussion by mstrolo** [Ysurac/openmptcprouter#2302] [Q&A]
Hi.

I was wondering... all the other solutions such as Peplink Speedify and such are they working with the same technology?
Mptcp and the surrounding?

**Comment by Ysurac** [comment]
No. They are based on closed source VPN, I think they work like Glorytun UDP.

**Reply by mstrolo** [reply]
can you explain better? 
how do they work how Mptcp work?
I know Bondix is working something like a backup client with a server host on the other side something like that

**Reply by Ysurac** [reply]
I never tested them and they are closed sources solutions, I don't know how they work.
For MPTCP docs: http://multipath-tcp.org/
